client.js

```diff
@@ -217,20 +206,49 @@ async function selectCountriesAndCities(hzClient) {
     console.log("--------------------");
 }
 
+async function nonStopMapExample(map) {
+    console.log("Now, `map` will be filled with random entries.");
+
+    let iterationCounter = 0;
+    while (true) {
+        const randomKey = Math.floor(Math.random() * 100000);
+        await map.put('key' + randomKey, 'value' + randomKey);
+        await map.get('key' + randomKey);
```

Comment:
This logic is not the same as Java's logic. You should call two Math.random() in order to match Java.